src/main/java/org/takes/rq/form/RqFormBase.java

```diff
@@ -168,7 +163,7 @@ private static String decode(final CharSequence txt) {
                 );
             }
             final String key = decode(
-                new EnglishLowerCase(parts[0].trim()).string()
+                new Utf8String(parts[0].trim()).englishLowerCase()
```

Comment:
@dalifreire looks like after your changes we'll have two ways of creating lowercase strings - either using dedicated class or Utf8String...